### DISCUSSION TITLE
Docker major upgrades only when latest changed

### DISCUSSION THREAD

**Discussion by Kiina** [renovatebot/renovate#22145] [Suggest an Idea]
**What would you like Renovate to be able to do?**

This might actually be kinda simple. I always enable the docker major upgrades so I get a notification when a newer version is out. Let's take mariadb for this example. 10.3 is the latest tag and the stable version. But dockerhub also lists the 10.4 RC. So renovate tells me: Hey, there is a new version 10.4. But I obviously don't want the RC version in production. So you need an option that checks and only tells you about the 10.4 version, as soon as that one is tagged as latest on dockerhub. You can also look at mongodb. They tag 4.1 as unstable and 4.0 as latest. So I only want a notification for 4.1 as soon as it's considered stable by the team and switched over to the latest tag.

**Additional context**

![image](https://user-images.githubusercontent.com/2419591/55336951-14ada100-549e-11e9-8c82-061771b845c2.png)

**Comment by rarkins** [comment]
I see what you mean, but the implementation will be challenging. 

Docker doesn’t expose any nice relational API that says “10.3 is the latest”. I think we’d need to start pulling the sha256 of every release until we hit one that’s same as latest, and also handle the case where none is found (it can happen by accident).

**Comment by rarkins** [comment]
We need to determine if we have any way to correlate that "latest" tag ourselves.
